### DISCUSSION TITLE
Queries regarding experiment output

### DISCUSSION THREAD

**Discussion by ghost** [pybamm-team/PyBaMM#2271] [Q&A] [answered]
[Untitled.pdf](https://github.com/pybamm-team/PyBaMM/files/9442377/Untitled.pdf)
I'm facing some problems regarding the experiment simulation. I'm new to this programing and can't crack it. Please let me know where I'm writing wrong code? Thanks in advance

**Comment by valentinsulzer** [comment, marked as answer]
You should ask this in liionpack
